# List of solved problems

	[.]	ID		Title
	[x]	DNA		Counting DNA Nucleotides
	[x]	RNA		Transcribing DNA into RNA
	[x]	REVC	Complementing a Strand of DNA
	[x]	GC		Computing GC Content
	[x]	HAMM	Counting Point Mutations
	[x]	SUBS	Finding a Motif in DNA
	[x]	CONS	Consensus and Profile
	[x]	GRPH	Overlap Graphs
	[x]	IPRB	Mendel's First Law
	[x]	LCS		Finding a Shared Motif
	[x]	PERM	Enumerating Gene Orders
	[ ]	PROT	Protein Translation
	[ ]	REVP	Locating Restriction Sites
	[ ]	IEV		Calculating Expected Offspring
	[ ]	LEXF	Enumerating k-mers Lexicographically
	[ ]	LIA		Independent Alleles
	[ ]	LONG	Genome Assembly as Shortest Superstring
	[ ]	MPRT	Finding a Protein Motif
	[ ]	MRNA	Inferring mRNA from Protein
	[ ]	ORF		Open Reading Frames
	[ ]	PROB	Introduction to Random Strings
	[ ]	PRTM	Calculating Protein Mass
	[ ]	REAR	Reversal Distance
	[ ]	SIGN	Enumerating Oriented Gene Orderings
	[ ]	SPLC	RNA Splicing
	[ ]	TREE	Completing a Tree
	[ ]	CORR	Error Correction in Reads
	[ ]	INOD	Counting Phylogenetic Ancestors
	[ ]	KMER	k-Mer Composition
	[ ]	KMP		Speeding Up Motif Finding
	[ ]	LEXV	Ordering Strings of Varying Length Lexicographically
	[ ]	PDST	Creating a Distance Matrix
	[ ]	PPER	Partial Permutations
	[ ]	RSTR	Matching Random Motifs
	[ ]	SORT	Sorting by Reversals
	[ ]	SSEQ	Finding a Spliced Motif
	[ ]	SSET	Counting Subsets
	[ ]	TRAN	Transitions and Transversions
	[ ]	ASPC	Introduction to Alternative Splicing
	[ ]	EVAL	Expected Number of Restriction Sites
	[ ]	LCSQ	Finding a Shared Spliced Motif
	[ ]	LREP	Finding the Longest Multiple Repeat
	[ ]	NWCK	Distances in Trees
	[ ]	SETO	Introduction to Set Operations
	[ ]	SPEC	Inferring Protein from Spectrum
	[ ]	CONV	Comparing Spectra with the Spectral Convolution
	[ ]	CTBL	Creating a Character Table
	[ ]	DBRU	Constructing a De Bruijn Graph
	[ ]	EDIT	Edit Distance
	[ ]	FULL	Inferring Peptide from Full Spectrum
	[ ]	INDC	Independent Segregation of Chromosomes
	[ ]	NKEW	Newick Format with Edge Weights
	[ ]	RNAS	Counting RNA Secondary Structures
	[ ]	SUFF	Encoding Suffix Trees
	[ ]	AFRQ	Counting Disease Carriers
	[ ]	CSTR	Creating a Character Table from Genetic Strings
	[ ]	CUNR	Counting Unrooted Binary Trees
	[ ]	EDTA	Edit Distance Alignment
	[ ]	LING	Linguistic Complexity of a Genome
	[ ]	MREP	Identifying Maximal Repeats
	[ ]	PCOV	Genome Assembly with Perfect Coverage
	[ ]	PRSM	Matching a Spectrum to a Protein
	[ ]	QRT		Quartets
	[ ]	SGRA	Using the Spectrum Graph to Infer Peptides
	[ ]	CHBP	Character-Based Phylogeny
	[ ]	CNTQ	Counting Quartets
	[ ]	GASM	Genome Assembly Using Reads
	[ ]	GLOB	Global Alignment with Scoring Matrix
	[ ]	MEND	Inferring Genotype from a Pedigree
	[ ]	ROOT	Counting Rooted Binary Trees
	[ ]	SEXL	Sex-Linked Inheritance
	[ ]	SPTD	Phylogeny Comparison with Split Distance
	[ ]	WFMD	The Wright-Fisher Model of Genetic Drift
	[ ]	ASMQ	Assessing Assembly Quality with N50 and N75
	[ ]	CSET	Fixing an Inconsistent Character Set
	[ ]	FOUN	The Founder Effect and Genetic Drift
	[ ]	GCON	Global Alignment with Constant Gap Penalty
	[ ]	GREP	Genome Assembly with Perfect Coverage and Repeats
	[ ]	LOCA	Local Alignment with Scoring Matrix
	[ ]	MGAP	Maximizing the Gap Symbols of an Optimal Alignment
	[ ]	MULT	Multiple Alignment
	[ ]	QRTD	Quartet Distance
	[ ]	ALPH	Alignment-Based Phylogeny
	[ ]	GAFF	Global Alignment with Scoring Matrix and Affine Gap Penalty
	[ ]	LAFF	Local Alignment with Affine Gap Penalty
	[ ]	RSUB	Identifying Reversing Substitutions